frontend/src/contexts/user-context.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
+
+import { createContext, useContext, ReactNode, useState } from "react";
+
+interface UserContextType {
+  user: User | null;
+  setUser: React.Dispatch<React.SetStateAction<User | null>>;
+}
+
+const initialUser: User | null = null;
+
+/* eslint-disable @typescript-eslint/no-empty-function */
```

Comment:
Instead of disabling the check, we can initialize the function to throw a not implemented error. This is clearer and align with our maintainability checks

Reply:
Resolved